injected/integration-test/scriptlets.spec.js

```diff
@@ -0,0 +1,636 @@
+import { gotoAndWait, testContextForExtension } from './helpers/harness.js';
+import { test as base, expect } from '@playwright/test';
+
+const test = testContextForExtension(base);
+
+test.describe('Scriptlets Integration Tests', () => {
```

Comment:
Is all this a dupe of the test-pages stuff? I see a lot of waitForTimeout too which is pretty flakey.